packages/apps/web-viewer-test/package.json

```diff
@@ -38,18 +37,15 @@
     "redux": "^4.0.5"
   },
   "devDependencies": {
-    "@svgr/webpack": "4.3.3",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^10.4.4",
-    "@testing-library/user-event": "^7.1.2",
+    "@bentley/react-scripts": "^4.0.3-dev.1",
+    "@svgr/webpack": "^5.5.0",
     "@types/history": "^4.7.6",
-    "@types/jest": "^26.0.4",
     "@types/node": "^14.0.19",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.5",
-    "typescript": "^3.7.2"
+    "typescript": "~4.1.0"
```

Comment:
i didnt bump the ts ver across the repo, i could if you like, but this is the ver core is now using and thought itd be help to stay on the same ver